CinemaCat.Api/appsettings.Development.json

```diff
@@ -5,9 +5,11 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "MongoDB":
-  {
+  "MongoDB": {
     "ConnectionString": "mongodb://user:password@localhost:27017/",
```

Comment:
probably it's better to move the connection string from this section to the ConnectionStrings